openpgp/internal/ecc/curve_info.go

```diff
@@ -93,3 +102,39 @@ func FindByOid(oid encoding.Field) *CurveInfo {
 	}
 	return nil
 }
+
+func FindEdDSAByType(curveType string) EdDSACurve {
```

Comment:
And then here, I know that this is consistent with the function above but I find (:upside_down_face:) this name to be a bit confusing as well, I would name it `EdDSACurveByGenName`, for example (and similarly for the following ones)